src/main/java/dk/cngroup/lentils/controller/ProgressController.java

```diff
@@ -102,12 +102,27 @@ public String teamProgress(final @RequestParam("teamId") Long teamId, final Mode
     @GetMapping(value = "/stage")
     public String stageProgress(final @RequestParam("cypherId") Long cypherId,
                                 final @RequestParam(value = "search", required = false) String searchString,
+                                final @RequestParam(value = "viewOtherTeams", required = false) Boolean viewOtherTeams,
```

Comment:
Kdyz to pouzijes takto:
```java
@RequestParam(value = "viewOtherTeams", required = false, default = "false") Boolean viewOtherTeams
```
Tak nemusis kontrolovat zda je `viewOtherTeams` null.

Reply:
good point :) upravim